addons/account/models/account_move.py

```diff
@@ -151,6 +151,8 @@ def _get_default_invoice_incoterm(self):
         string='Partner', change_default=True)
     commercial_partner_id = fields.Many2one('res.partner', string='Commercial Entity', store=True, readonly=True,
         compute='_compute_commercial_partner_id')
+    country_code = fields.Char(string='Country Code', readonly=True,
+        related='company_id.country_id.code', change_default=True)
```

Comment:
change_default?

Reply:
string= not necessary I think (readonly=True also not, but it can not hurt to be more precise)